arch/arm/cpu/armv8/start.S

```diff
@@ -16,19 +16,27 @@
  *
  *************************************************************************/
 
-.globl	_start
-_start:
 #if defined(CONFIG_LINUX_KERNEL_IMAGE_HEADER)
+
```

Comment:
Extra whitespace changes all around this patch. It makes review process difficult. 

Please refrain from changes that are not related to intended patch purpose.

Reply:
Hi @lorc, I guess you mean "extra new lines", and not the extra "white spaces".

I've removed them. Done.

But, in general, from my point of view, it makes the code more readable in this case. I still believe in this. But you are the maintainer, so I obey. ))